```
Python/ceval_gil.c: In function ‘create_gil’:
Python/ceval_gil.c:188:5: error: implicit declaration of function ‘_Py_ANNOTATE_RWLOCK_CREATE’ [-Werror=implicit-function-declaration]
     _Py_ANNOTATE_RWLOCK_CREATE(&gil->locked);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~
Python/ceval_gil.c: In function ‘destroy_gil’:
Python/ceval_gil.c:204:5: error: implicit declaration of function ‘_Py_ANNOTATE_RWLOCK_DESTROY’ [-Werror=implicit-function-declaration]
     _Py_ANNOTATE_RWLOCK_DESTROY(&gil->locked);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
Python/ceval_gil.c: In function ‘drop_gil’:
Python/ceval_gil.c:243:5: error: implicit declaration of function ‘_Py_ANNOTATE_RWLOCK_RELEASED’ [-Werror=implicit-function-declaration]
     _Py_ANNOTATE_RWLOCK_RELEASED(&gil->locked, /*is_write=*/1);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
Python/ceval_gil.c: In function ‘take_gil’:
Python/ceval_gil.c:356:5: error: implicit declaration of function ‘_Py_ANNOTATE_RWLOCK_ACQUIRED’ [-Werror=implicit-function-declaration]
     _Py_ANNOTATE_RWLOCK_ACQUIRED(&gil->locked, /*is_write=*/1);
```

Okay we catch one.

--- a/Modules/_testcapi/pyatomic.c
+++ b/Modules/_testcapi/pyatomic.c
@@ -145,8 +145,7 @@ test_atomic_load_store_int_release_acquire(PyObject *self, PyObject *obj) { \
     int x = 0;
     int y = 1;
     int z = 2;
-    _Py_atomic_load_int_acquire(&x);
-    assert(x == 0);
+    assert(_Py_atomic_load_int_acquire(&x) == 0);
     _Py_atomic_store_int_release(&x, y);
     assert(x == y);
     assert(_Py_atomic_load_int_acquire(&x) == y);

--- a/Python/ceval_gil.c
+++ b/Python/ceval_gil.c
@@ -1,5 +1,6 @@
 
 #include "Python.h"
+#include "pycore_atomic.h"        // _Py_ANNOTATE_RWLOCK_XXXX
 #include "pycore_ceval.h"         // _PyEval_SignalReceived()
 #include "pycore_initconfig.h"    // _PyStatus_OK()
 #include "pycore_interp.h"        // _Py_RunGC()

--- a/Python/thread_pthread.h
+++ b/Python/thread_pthread.h
@@ -1,5 +1,6 @@
 #include "pycore_interp.h"        // _PyInterpreterState.threads.stacksize
 #include "pycore_pythread.h"      // _POSIX_SEMAPHORES
+#include "pycore_atomic.h"        // _Py_ANNOTATE_RWLOCK_XXXX
 
 /* Posix threads interface */
 

